### PR TITLE
Release/ui

### PR DESCRIPTION
**Pull request by rolandvarga**

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -66,79 +65,105 @@ func (a *App) getAlerts(w http.ResponseWriter, r *http.Request) {
...
+		return "schedule repeat is empty", false
+	}
+
+	if r.Count <= scheduleCountMin {
+		return fmt.Sprintf("repeat count must be higher than %d", scheduleCountMin), false
 	}
-	if r.Count > 50 {
-		return false
+
+	if r.Count >= scheduleCountMax {
```
Comment (louisfelix) [inline app.go]: As per returned text:
```suggestion
	if r.Count > scheduleCountMax {
```

File: templates/layout.gohtml [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+                                <input type="text" class="form-control" name="CreatedBy" id="createdBy" required/>
+                            </div>
+                        </div>
+
+                        {{ range .Flashes }}
+                            <div class="alert alert-{{.Status}} mt-4">{{ .Message }}</div>
+                        {{ end }}
+
+                        <div class="alert alert-warning mt-4 collapse" role="alert" id="alertMessage">
+                            You forgot something
```
Comment (louisfelix) [inline templates/layout.gohtml]: Is there a more precise way to formulate this?

Comment (rolandvarga) [reply templates/layout.gohtml]: @louisfelix that div is hidden by default and the message never gets displayed. As soon as there is a warning the contents of the div are updated and made visible. I could replace with an empty string if you prefer that.

Comment (louisfelix) [reply templates/layout.gohtml]: Ah yes. OK empty string!

File: app.go (hunk 1 above)
Comment (rolandvarga) [reply app.go]: changed

File: templates/layout.gohtml (hunk 2 above)
Comment (rolandvarga) [reply templates/layout.gohtml]: changed

File: README.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+```
+
+### Configuration & Running
+The tool relies on a YAML config file to specify the Alertmanager address it is supposed to send requests to:
+```yaml
+---
+alertmanager_api: "http://localhost:9093/api/v2"
+```
+
+It is required to setup the `SESSION_KEY` environment variable with a hash key, that is used authenticate cookie values via HMAC.
```
Comment (louisfelix) [inline README.md]: What needs to be done exactly here?

Comment (rolandvarga) [reply README.md]: I have updated the docs. Let me know what you think.

Comment (louisfelix) [reply README.md]: ok!

File: templates/layout.gohtml [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+                showAlert("Start time of Maintenance window must be specified.");
+                return false;
+            }
+            let endTime = document.getElementById("endTime").value;
+            if (endTime === "") {
+                showAlert("End time of Maintenance window must be specified.");
+                return false;
+            }
+
+            if (startTime === endTime) {
```
Comment (louisfelix) [inline templates/layout.gohtml]: `startTime` can still be "after" `endTime` here.

File: templates/layout.gohtml [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+                showAlert("End time must be higher than start time.");
```
Comment (louisfelix) [inline templates/layout.gohtml]:
```suggestion
                showAlert("End time must be after start time.");
```

File: templates/layout.gohtml (hunk 4 above)
Comment (rolandvarga) [reply templates/layout.gohtml]: added another condition

File: templates/layout.gohtml (hunk 5 above)
Comment (rolandvarga) [reply templates/layout.gohtml]: I chose a 3rd option `later than`

**Pull request merged**